notebooks/network_mapping/Supervised_Asset_Classification.ipynb

```diff
@@ -0,0 +1,1600 @@
+{
```

Comment:
<p>Can you talk a bit more about the optimizer? What are you using? Perhaps writing out the formula in markdown would be useful too.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/clx/pull/193/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>